sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -839,6 +856,36 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
     if (isUpdate) {
       webResource.headers.set("If-Match", "*");
     }
+
+    const forwardTo = (entityFields as InternalQueueOptions | InternalSubscriptionOptions)
+      .ForwardTo;
+
+    const forwardDeadLetterMessagesTo = (entityFields as
+      | InternalQueueOptions
+      | InternalSubscriptionOptions).ForwardDeadLetteredMessagesTo;
+
+    if (forwardTo) {
+      webResource.headers.set(
+        "ServiceBusSupplementaryAuthorization",
+        (await this.sasTokenProvider.getToken(this.endpoint)).token
+      );
+
+      (entityFields as InternalQueueOptions | InternalSubscriptionOptions).ForwardTo =
+        "https://" + this.endpoint + "/" + forwardTo;
```

Comment:
Can we consider `forwardTo` and `forwardDeadLetterMessagesTo` properties to be absolute urls when provided by the user?

The service expects it to be an absolute url and also returns the responses with these properties set to be absolute urls.

Reply:
Service expects to be on the same namespace in general, since we retrieve authorization tokens for the given connection string.
Also, current behavior is so it can be consistent with .NET SDK where we accept only the path and not the entire URL

Reply:
Looked into .NET SDK more closely, and looks like the URLs actually use a `sb://` prefix.
Updated to reflect the same.
Also figured rootcause to why ForwardTo wasn't getting updated.
Basically changes as part of https://github.com/Azure/azure-sdk-for-js/pull/6395 touch these parameters and considerations as well.

Reply:
Can you please provide a pointer to the .Net code which we are trying to emulate here? Same for the part where we convert the service returned value back to relative path.

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/QueueDescriptionExtensions.cs#L208

Couldn't spot where the returned value is parsed to include only the relative path, which seems to be erroneous if true.

Reply:
My C# is a little spotty, but based on the code linked above, it looks like absolute urls are indeed supported. Only if the user provided value is not an absolute url, the endpoint is used to create an absolute url.

But, this PR assumes that the user always provides a relative url.

> Couldn't spot where the returned value is parsed to include only the relative path, which seems to be erroneous if true.

Have we run the .Net SDK and checked whether the response for `forwardTo` contains absolute urls or relative urls?

Reply:
Based on this sample, looks like passing in the relative path is supported and usual? https://github.com/Azure/azure-service-bus/tree/master/samples/DotNet/Microsoft.ServiceBus.Messaging/AutoForward

Reply:
Also, yes - looks like supporting absolute URI as well is what JS may need to do as well, thanks for the catch!

Reply:
Updated to handle as absolute URIs by default, while accepting relative paths as input optionally